contracts/strategies/_poc/easy-rf/EasyRetroFundingStrategy.sol

```diff
@@ -0,0 +1,737 @@
+
+    /// @notice Emitted when a recipient updates their registration
+    /// @param recipientId Id of the recipient
+    /// @param data The encoded data - (address recipientId, address recipientAddress, Metadata metadata)
+    /// @param sender The sender of the transaction
+    /// @param status The updated status of the recipient
+    event UpdatedRegistration(address indexed recipientId, bytes data, address sender, uint8 status);
+
+    /// @notice Emitted when a recipient is registered and the status is updated
+    /// @param rowIndex The index of the row in the bitmap
+    /// @param fullRow The value of the row
+    /// @param sender The sender of the transaction
+    event RecipientStatusUpdated(uint256 indexed rowIndex, uint256 fullRow, address sender);
+
+    /// @notice Emitted when the timestamps are updated
+    /// @param registrationStartTime The start time for the registration
+    /// @param registrationEndTime The end time for the registration
+    /// @param poolStartTime The start time
+    /// @param poolEndTime The end time
+    /// @param sender The sender of the transaction
+    event TimestampsUpdated(
+        uint64 registrationStartTime,
+        uint64 registrationEndTime,
+        uint64 poolStartTime,
+        uint64 poolEndTime,
+        address sender
+    );
+
+    /// @notice Emitted when the distribution has been updated with a new metadata
+    /// @param metadata The metadata of the distribution
+    event DistributionUpdated(Metadata metadata);
+
+    /// @notice Emitted when funds are distributed to a recipient
+    /// @param amount The amount of tokens distributed
+    /// @param grantee The address of the recipient
+    /// @param token The address of the token
+    /// @param recipientId The id of the recipient
+    event FundsDistributed(uint256 amount, address grantee, address indexed token, address indexed recipientId);
+
+    /// @notice Emitted when a batch payout is successful
+    /// @param sender The sender of the transaction
+    event BatchPayoutSuccessful(address indexed sender);
+
+    /// ================================
+    /// ========== Storage =============
+    /// ================================
+
+    /// @notice Metadata containing the distribution data.
+    Metadata public distributionMetadata;
+
+    /// @notice Flag to indicate whether to use the registry anchor or not.
+    bool public useRegistryAnchor;
+
+    /// @notice Flag to indicate whether metadata is required or not.
+    bool public metadataRequired;
+
+    /// @notice Flag to indicate whether the distribution has started or not.
+    bool public distributionStarted;
+
+    /// @notice The timestamps in seconds for the start and end times.
+    uint64 public registrationStartTime;
+    uint64 public registrationEndTime;
+    uint64 public poolStartTime;
+    uint64 public poolEndTime;
+
+    /// @notice The total amount of tokens allocated to the payout.
+    uint256 public totalPayoutAmount;
```

Comment:
this is being used instead of fundPool right ?

Reply:
not used anywhere, got removed